radish34/api/src/baseline/core/integrations/messenger/sendMessage/processResponse.js

```diff
@@ -1,6 +1,6 @@
 export default (job, done) => {
   try {
-    done(null, { message: 'API: Process Response', jobData: job.data });
+    done(null, { message: 'API: Process Response', jobData: data });
```

Comment:
Does this work as expected? It seems like `data` would be `undefined` unless it's destructured from the `job` object